src/styles/frontend-shared.scss

```diff
@@ -32,3 +32,29 @@
     min-height: auto;
   }
 }
+
+// Used in the AdderToolbar, a transparent button that has an icon and a small
+// label below the icon
+.LabeledIconButton {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
```

Comment:
One of the strategies right now is to remove use and reliance upon local mixins, so these staged extracted patterns have more rules unrolled than their pattern-ized brethren in `frontend-shared`. This is by plan.